openstates/md/bills.py

```diff
@@ -165,10 +170,132 @@ def parse_bill_votes(self, doc, bill):
                 vote.add_source(href)
                 yield vote
 
+    def parse_bill_votes_new(self, doc, bill):
+        elems = doc.xpath('//a')
```

Comment:
This `//a` query is very broad; can it be better-scoped?

Reply:
Okay, I did this one to the table containing bill docs.

Reply:
Okie dokie